commands.yaml

```diff
@@ -1,39 +1,42 @@
 help:
 exit:
 quit:
-aad login:
+o365 login:
   - '-t, --authType [authType]'
   - '-u, --userName [userName]'
   - '-p, --password [password]'
   - '-c, --certificateFile [certificateFile]'
   - '--thumbprint [thumbprint]'
-aad connect:
+o365 connect:
   - '-t, --authType [authType]'
   - '-u, --userName [userName]'
   - '-p, --password [password]'
   - '-c, --certificateFile [certificateFile]'
   - '--thumbprint [thumbprint]'
-aad logout:
-aad disconnect:
-aad oauth2grant add:
+o365 logout:
+o365 disconnect:
+o365 oauth2grant add:
```

Comment:
`oauth2grant` is ran on `aad` which is a service, so it should stay as-is

Reply:
Fully agree, I must have missed that.